src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -479,4 +483,29 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
       },
     },
   },
+  'Prompt-to-edit': {
+    description: 'Use Zoo AI to edit your kcl',
+    icon: 'chat',
+    args: {
+      selection: {
+        inputType: 'selection',
+        selectionTypes: [
+          'solid2D',
+          'segment',
+          'sweepEdge',
+          'cap',
+          'wall',
+          'edgeCut',
+          'edgeCutEdge',
+        ],
```

Comment:
Might be good to have a all kinda thing here because any selection is valid for submitting to the AI.